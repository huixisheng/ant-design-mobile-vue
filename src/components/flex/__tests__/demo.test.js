import webDemoTest from '../../../tests/shared/demoTest';

webDemoTest('flex');
