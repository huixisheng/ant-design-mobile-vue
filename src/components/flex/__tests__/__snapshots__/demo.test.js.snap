// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ./components/flex/demo/basic.md correctly 1`] = `
<div
  class="flex-container"
>
  <div
    class="sub-title"
  >
    Basic
  </div>
  <div
    class="am-flexbox am-flexbox-align-center"
  >
    <div
      class="am-flexbox-item"
    >
      <div
        class=" placeholder"
      >
        Block
      </div>
    </div>
    <div
      class="am-flexbox-item"
    >
      <div
        class=" placeholder"
      >
        Block
      </div>
    </div>
  </div>
  <div
    class="am-whitespace am-whitespace-lg"
  />
  <div
    class="am-flexbox am-flexbox-align-center"
  >
    <div
      class="am-flexbox-item"
    >
      <div
        class=" placeholder"
      >
        Block
      </div>
    </div>
    <div
      class="am-flexbox-item"
    >
      <div
        class=" placeholder"
      >
        Block
      </div>
    </div>
    <div
      class="am-flexbox-item"
    >
      <div
        class=" placeholder"
      >
        Block
      </div>
    </div>
  </div>
  <div
    class="am-whitespace am-whitespace-lg"
  />
  <div
    class="am-flexbox am-flexbox-align-center"
  >
    <div
      class="am-flexbox-item"
    >
      <div
        class=" placeholder"
      >
        Block
      </div>
    </div>
    <div
      class="am-flexbox-item"
    >
      <div
        class=" placeholder"
      >
        Block
      </div>
    </div>
    <div
      class="am-flexbox-item"
    >
      <div
        class=" placeholder"
      >
        Block
      </div>
    </div>
    <div
      class="am-flexbox-item"
    >
      <div
        class=" placeholder"
      >
        Block
      </div>
    </div>
  </div>
  <div
    class="am-whitespace am-whitespace-lg"
  />
  <div
    class="sub-title"
  >
    Wrap
  </div>
  <div
    class="am-flexbox am-flexbox-wrap am-flexbox-align-center"
  >
    <div
      class="inline placeholder"
    >
      Block
    </div>
    <div
      class="inline placeholder"
    >
      Block
    </div>
    <div
      class="inline placeholder"
    >
      Block
    </div>
    <div
      class="inline placeholder"
    >
      Block
    </div>
    <div
      class="inline placeholder"
    >
      Block
    </div>
    <div
      class="inline placeholder"
    >
      Block
    </div>
    <div
      class="inline placeholder"
    >
      Block
    </div>
  </div>
  <div
    class="am-whitespace am-whitespace-lg"
  />
  <div
    class="sub-title"
  >
    Align
  </div>
  <div
    class="am-flexbox am-flexbox-justify-center am-flexbox-align-center"
  >
    <div
      class="inline placeholder"
    >
      Block
    </div>
    <div
      class="inline placeholder"
    >
      Block
    </div>
    <div
      class="inline placeholder"
    >
      Block
    </div>
  </div>
  <div
    class="am-whitespace am-whitespace-md"
  />
  <div
    class="am-flexbox am-flexbox-justify-end am-flexbox-align-center"
  >
    <div
      class="inline placeholder"
    >
      Block
    </div>
    <div
      class="inline placeholder"
    >
      Block
    </div>
    <div
      class="inline placeholder"
    >
      Block
    </div>
  </div>
  <div
    class="am-whitespace am-whitespace-md"
  />
  <div
    class="am-flexbox am-flexbox-justify-between am-flexbox-align-center"
  >
    <div
      class="inline placeholder"
    >
      Block
    </div>
    <div
      class="inline placeholder"
    >
      Block
    </div>
    <div
      class="inline placeholder"
    >
      Block
    </div>
  </div>
  <div
    class="am-whitespace am-whitespace-md"
  />
  <div
    class="am-flexbox am-flexbox-align-start"
  >
    <div
      class="inline placeholder"
    >
      Block
    </div>
    <div
      class="inline small placeholder"
    >
      Block
    </div>
    <div
      class="inline placeholder"
    >
      Block
    </div>
  </div>
  <div
    class="am-whitespace am-whitespace-md"
  />
  <div
    class="am-flexbox am-flexbox-align-end"
  >
    <div
      class="inline placeholder"
    >
      Block
    </div>
    <div
      class="inline small placeholder"
    >
      Block
    </div>
    <div
      class="inline placeholder"
    >
      Block
    </div>
  </div>
  <div
    class="am-whitespace am-whitespace-md"
  />
  <div
    class="am-flexbox am-flexbox-align-baseline"
  >
    <div
      class="inline placeholder"
    >
      Block
    </div>
    <div
      class="inline small placeholder"
    >
      Block
    </div>
    <div
      class="inline placeholder"
    >
      Block
    </div>
  </div>
</div>
`;
